packages/editor/src/lib/editor/managers/WatermarkManager.test.ts

```diff
@@ -1,149 +0,0 @@
-import { createTLStore } from '../../config/createTLStore'
```

Comment:
one of the nice things about keeping this separate is that we were able to make this testable. we don't want to inadvertently make the watermark show up when you've actually paid (and vice-versa we want to make sure it shows up when it should), it could be very easy to mess this up!

recommendation: we pull out the <Watermark /> react component into a separate file so that we can easily add these tests back in and make sure we're doing the correct thing.

Reply:
That makes sense!